modules/Microsoft.AAD/DomainServices/deploy.bicep

```diff
@@ -57,13 +57,15 @@ param ntlmV1 string = 'Enabled'
   'Enabled'
   'Disabled'
 ])
+#disable-next-line secure-secrets-in-params
```

Comment:
Can we have a small paragraph in the module design wiki for linter and what we intend to ignore?

Reply:
Can do. I guess we'll need to revisit it every other month as the PG keeps on adding rules

Reply:
Not sure if in this or a separated PR, probably we would need to structure this a little more, but I'd like to hear your thoughts. 

We customize Bicep linter settings on 2 levels:

- The bicepconfig.json file for which we have a dedicated wiki page here https://github.com/Azure/ResourceModules/wiki/The%20CI%20environment%20-%20Bicep%20configuration%20file
- The single line to disable false positives as introduced in this PR (and also similarly leveraged by other 2 PRs currently in review) as documented here https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter#silencing-false-positives, a page we should reference since the list of rules is constantly updated as you correctly pointed out. 

So in the Wiki we should either have them in one page or at least reference each other. I find it a bit confusing that the first is documented in the CI environment while the second is documented in the module design.

For the latter, as also suggested in the link above, we should `add a comment explaining why the rule does not apply to this line.` in each .bicep file.

Reply:
They are in fact about 2 distinct use cases that are only connected by the fact the the `bicepconfig.json` file is able to configure a subset of linter rules.

That being said, I'll go ahead and merge the two regardless - and to do that rename the original docs page to not reference the 'bicep config file' at all, but rather a 'bicep config' in general.

Reply:
done

Reply:
I'd still suggest to `add a comment explaining why the rule does not apply to this line` to each .bicep file where we are silencing false positives, as mentioned here https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter#silencing-false-positives

Reply:
Done :)